ebmbot/bot.py

```diff
@@ -56,7 +56,8 @@ def register_listeners(app, config, channels, bot_user_id):
     """
     Register listeners for this app
 
-    - A single listener for job requests
+    - A single listener for job requests that mention the bot
+    - A single listener for job requests in DMs with the bot
```

Comment:
It's just occurred to me that "job request" might be a little confusing in our context… would "task request" still work?

Reply:
Hah yes, another layer to the `job` meaning.  Yes, "task request" would work here, but it's "job" because it refers to something that's in the job_config, so that should probably also be updated. Although, this covers things like `@bot status` and `@bot help` which are requests from slack but not jobs/tasks, so maybe "slack requests" is OK to cover all requests.